seq2seq_model.py

```diff
@@ -140,11 +140,18 @@ def sampled_loss(inputs, labels):
             dtype)
       softmax_loss_function = sampled_loss
     # Create the internal multi-layer cell for our RNN.
-    single_cell = tf.nn.rnn_cell.GRUCell(size)
-    if use_lstm:
-      single_cell = tf.nn.rnn_cell.BasicLSTMCell(size)
 
-    cell = single_cell
+    # Create the internal multi-layer cell for our RNN.
+    if use_lstm:
+      if num_layers > 1 :
+        cell = tf.contrib.rnn.MultiRNNCell([tf.contrib.rnn.BasicLSTMCell(size) for _ in range(num_layers)])
+      else :
+        cell = tf.contrib.rnn.BasicLSTMCell(size)
```

Comment:
Can you name this variable something like `state` instead of `cell`, since it's not a single cell anymore?